desktop/main.js

```diff
@@ -323,7 +340,16 @@ const mainWindow = () => {
                             {id: 'cut', role: 'cut'},
                             {id: 'copy', role: 'copy'},
                             {id: 'paste', role: 'paste'},
-                            {id: 'pasteAndMatchStyle', role: 'pasteAndMatchStyle'},
+                            {
+                                id: 'pasteAndMatchStyle',
+                                role: 'pasteAndMatchStyle',
+                                accelerator: DESKTOP_SHORTCUT_ACCELERATOR.PASTE_AND_MATCH_STYLE,
```

Comment:
Why do we need to add the accelerator here?

Reply:
Apologies, as per my testing, shortcuts that are defined in the App Menu work, while the shortcuts defined in contextMenu dont. Their only purpose seems to show their value next to their name

Reply:
Thoretically, from what i understand, contextMenu shortcuts are meant to work only when contextMenu is open. However, practically, if you press any key, contextMenu is gone and the shortcut, thus, won't work